feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/section/FloorMap.kt

```diff
@@ -0,0 +1,63 @@
+package io.github.droidkaigi.confsched2023.floormap.section
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import io.github.droidkaigi.confsched2023.feature.floormap.R
+import io.github.droidkaigi.confsched2023.floormap.FloorMapStrings
+import io.github.droidkaigi.confsched2023.model.FloorLevel
+import io.github.droidkaigi.confsched2023.model.FloorLevel.Basement
+import io.github.droidkaigi.confsched2023.model.FloorLevel.Ground
+
+@Composable
+fun FloorMap(
+    floorLevel: FloorLevel,
+    modifier: Modifier = Modifier,
+) {
+    val headline = when (floorLevel) {
+        Basement -> FloorMapStrings.Basement.asString()
+        Ground -> FloorMapStrings.Ground.asString()
+    }
+    val mapResId = when (floorLevel) {
+        Basement -> R.drawable.img_floormap_basement
+        Ground -> R.drawable.img_floormap_ground
+    }
```

Comment:
It's up to you because I can create issue for this but you can create UiState for this and you might be able to remove this function.
https://github.com/DroidKaigi/conference-app-2023#section

Reply:
Ok. I'll try it.

Reply:
- [refactor: introduce FloorMapUiState](https://github.com/DroidKaigi/conference-app-2023/pull/551/commits/558925ff38327e4b02d5117f0c8acbadb7be1c45)
- [refactor(uiState): change uiState-factory method](https://github.com/DroidKaigi/conference-app-2023/pull/551/commits/0b23c8c733639cb8ee2014b8818107289357679d)

**What we did**
1. define FloorMapUiState
2. define FloorMapUiState factory method
3. add FloorMapUiState to FloorMapScreenUiState
4. create FloorMapUiState from FloorMapViewModel
5. remove duplicated string resources